examples/multipleInternals/src/lemmas.k.md

```diff
@@ -0,0 +1,19 @@
+
+```k
```

Comment:
These are already present in rules.k

Reply:
Cool, I will also remove them from `multipleCalls` (where they were copy-pasted from).